File: Problem2/Java/hyejinjeong/problem2.java
public class problem2 {
	public static void main(String[] args){
		int add = 0;
		int n1 = 0, n2 = 1;
		int n3 = n1 + n2;
		
		while(n3 <= 4000000){
			if(n3 % 2 == 0)
				add += n3;
			n3 = n1 + n2;
			n1 = n2;
			n2 = n3;
		}
		System.out.println(add);
	}
}

File: Problem2/Python/jungmin/Problem2.py
# 피보나치 수열에서 4백만 이하이면서 짝수인 항의 합


result = [1, 2]


def problem2():
    while True:
        num = result[-1] + result[-2]
        if num > 4000000: break
        else: result.append(num)
    return result

print(sum(i for i in problem2() if i % 2 == 0))
